### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoptions_ScreenReader : VoiceOver is not announcing the table name 

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7818]
.
GitHubTags: #SQL Azure Data Studio;#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#DesktopApp;#VoiceOver;#Mac;
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z

VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: MACOS MOJAVE Version 10.14.6(18G87)
**Additional Details:**
VoiceOver
**Steps to Reproduce :**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.     Start Voice over and observe the announcement.
**Actual**:
Voice over is not announcing the table name.
**Expected:**
Voice over should announce the table name.
**User Impact:**
Screen reader User will not able to relate with the table.
**Attachment for Reference:**
[A11y_AzureDataStudio_Selectthedeploymentoptions_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3742978/A11y_AzureDataStudio_Selectthedeploymentoptions_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by vinodbk197** [comment]
Verified this issue on:
Azure Data Studio
Version: 1.15.0-insider (user setup)

The issue is no longer to reproduce, Hence closing this issue.
[7818_Closed.pptx](https://github.com/microsoft/azuredatastudio/files/4138868/7818_Closed.pptx)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_Mac;

**Comment by sspatil123** [comment]
#A11ySev2;
